apps/src/templates/sectionProgressV2/InviteToV2ProgressModal.jsx

```diff
@@ -134,15 +134,6 @@ function InviteToV2ProgressModal({
           aria-label={i18n.dialogAnnouncement()}
           className={styles.dialog}
         >
-          <button
-            id="ui-close-dialog"
-            type="button"
-            onClick={handleDismiss}
-            aria-label={i18n.closeDialog()}
-            className={styles.xCloseButton}
-          >
-            <i id="x-close" className="fa-solid fa-xmark" />
-          </button>
```

Comment:
This modal makes a distinction between dismissing the dialog and closing the dialog and then logs those separate events for analytics. 
- [ ] check with Teacher Tools about this and see if implementing a workaround is needed